client/server.go

```diff
@@ -641,19 +682,32 @@ func (c *CqlServerConnection) invokeRequestHandlers(request *frame.Frame) {
 	c.waitGroup.Add(1)
 	go func() {
 		log.Debug().Msgf("%v: invoking request handlers for incoming request: %v", c, request)
-		var response *frame.Frame
 		var err error
-		for i, handler := range c.handlers {
-			if response = handler(request, c, c.handlerCtx[i]); response != nil {
-				log.Debug().Msgf("%v: request handler %v produced response: %v", c, i, response)
-				if err = c.Send(response); err != nil {
-					log.Error().Err(err).Msgf("%v: send failed for frame: %v", c, response)
+		var rawResponse []byte
+		for i, rawHandler := range c.rawHandlers {
+			if rawResponse = rawHandler(request, c, c.handlerCtx[i]); rawResponse != nil {
+				log.Debug().Msgf("%v: request handler %v produced response: %v", c, i, rawResponse)
+				if err = c.SendRaw(rawResponse); err != nil {
+					log.Error().Err(err).Msgf("%v: send failed for frame: %v", c, rawResponse)
 				}
 				break
 			}
 		}
-		if response == nil {
+		if rawResponse == nil {
 			log.Debug().Msgf("%v: no request handler could handle the request: %v", c, request)
```

Comment:
Hmm this message is now misleading, since we haven't tried yet the "normal" request handlers.

Reply:
I removed this message and kept the one after every handler type is attempted.